### PR TITLE
Rename SpeechRecognitionError to SpeechRecognitionErrorEvent

### PR DESCRIPTION
**Pull request by sideshowbarker**
https://github.com/WICG/speech-api/pull/23 renamed `SpeechRecognitionError` to `SpeechRecognitionErrorEvent`. (No engines other than Blink/Chromium yet implement it as `SpeechRecognitionErrorEvent`, and no others ever implemented `SpeechRecognitionError`.) https://github.com/chromium/chromium/commit/5dbd1fc 

---

cc @jpmedley for confirmation of the Chrome, etc., version numbers — and for additional feedback about whether the patch here is the right way to handle this kind of case.

### CODE REVIEW

**Comment by jpmedley** [comment]
The data looks correct, but don't we handle name changes with `"alternative_name"`? @Elchi3

**Comment by sideshowbarker** [comment]
I’m still planning to make the changes requested in https://github.com/mdn/browser-compat-data/pull/5584#pullrequestreview-350821673. I just haven’t been able to set aside the time yet — but I expect I will be able to get back to it this week.

**Comment by sideshowbarker** [comment]
> 1. Rename the file to the new name
> 2. Set an `alternative_name` to the old name to the existing compat statements
> 3. Add a `version_removed` to all browsers that have renamed it, and add a new statement with the `version_added` of when it was renamed

OK, I’ve now done all that, and pushed it in bd76222

File: api/SpeechRecognitionErrorEvent.json [hunk 1]
```diff
@@ -38,16 +50,26 @@
...
-            "prefix": "webkit",
-            "version_added": true,
-            "notes": "You'll need to serve your code through a web server for recognition to work."
-          }
+          "samsunginternet_android": [
+            {
+              "version_added": false
+            },
+            {
+              "alternative_name": "webkitSpeechRecognitionError",
```
Comment (vinyldarkscratch) [inline api/SpeechRecognitionErrorEvent.json]: I think we can remove the single `”version_added”: false` statement and just leave it to the one with the alternative name?

File: api/SpeechRecognitionErrorEvent.json [hunk 2]
```diff
@@ -1,21 +1,33 @@
...
+              "version_added": "33",
+              "version_removed": "77"
+            }
+          ],
           "edge": {
-            "version_added": null
+            "version_added": false
```
Comment (vinyldarkscratch) [inline api/SpeechRecognitionErrorEvent.json]: This should be “79”, along with all other Edge data.

File: api/SpeechRecognitionErrorEvent.json (hunk 1 above)
Comment (sideshowbarker) [reply api/SpeechRecognitionErrorEvent.json]: > I think we can remove the single `”version_added”: false` statement and just leave it to the one with the alternative name?

Yep, OK — done

File: api/SpeechRecognitionErrorEvent.json (hunk 2 above)
Comment (sideshowbarker) [reply api/SpeechRecognitionErrorEvent.json]: OK — changed

File: api/SpeechRecognitionErrorEvent.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/SpeechRecognitionErrorEvent.json]: Thank you!  We’ll also need to make it not an array.

Comment (sideshowbarker) [reply api/SpeechRecognitionErrorEvent.json]: > Thank you! We’ll also need to make it not an array.

oops, yup — done

**Pull request merged**